MVP/sample_job.yaml

```diff
@@ -0,0 +1,28 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
+    triggers:
+        - ci-trigger:
+            jms-selector: "name = '{component}' AND CI_TYPE = 'brew-tag' AND (tag LIKE 'RHEL-6._-candidate' OR tag LIKE 'rhel-7._-candidate')"
+    builders:
+        - shell: |
+            set -x
+
+            # Here goes your shell commands for submitting beaker job here, and save the job ID into env var BKR_JOBID
+            # E.g. `bkr workflow-tomorrow --distro=$distro --task-file=$taskfile $other_args`
+            BKR_JOBID="J:1000000"
+
+            bkr job-watch $BKR_JOBID
+            bkr job-results --format junit-xml $BKR_JOBID > $WORKSPACE/$BKR_JOBID.xml
```

Comment:
IIUC, we'll need users to highly unify their test submitting process first, in order to abstract the builder/shell portion into template as well.

It's gonna be a theoretical practice given teams are storing test metadata all around in different places in different format.